src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -83,6 +83,120 @@ public static class DnnCatalog
             return new DnnRetrainEstimator(env, options, DnnUtils.LoadDnnModel(env, modelPath, true));
         }
 
+        /// <summary>
+        /// The options for the <see cref="ImageClassificationTransformer"/>.
+        /// </summary>
+        public sealed class Options
+        {
+            /// <summary>
+            /// The names of the model inputs.
+            /// </summary>
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The names of the model inputs", ShortName = "inputs", SortOrder = 1)]
```

Comment:
Why is this marked with `ArgumentType.Multiple`? We only support a single `Features` column name, correct?